book/website/welcome.md

```diff
@@ -1,13 +1,20 @@
 (welcome)=
 # Welcome
 
-_The Turing Way_ is a handbook to reproducible, ethical and collaborative data science. 
+*Welcome to The Turing Way guides to reproducible, ethical and collaborative data science.*
 
-The project is open source (anyone can [inspect the code](https://github.com/alan-turing-institute/the-turing-way)), open contribution (anyone can {ref}`contribute<ch-contributing>`), and open licenced (anyone can [re-use the materials](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md)).
+_The Turing Way_ is an open source, open collaboration and community-driven project.
+We involve and support a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
+Our goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need at the start of their projects to ensure that they are easy to reproduce at the end.
 
-Our goal is to provide all the information that data scientists in academia, industry, government and the third sector need to ensure that the projects they work on are easy to reproduce and reuse.
+```{note}
+_The Turing Way_ is not meant to be read from start to finish. 
+Start with a concept, tool or method that you need now, in your current work. 
+Browse the different guides that make up the book, or use the search box to search for whatever you would like to learn about first.
+```
 
-Warning: Do not try to read the _The Turing Way_ from start to finish. Take a [buffet approach](https://mobile.twitter.com/rossdavism/status/1138441421143400453). Browse the different guides that make up the book, or use the search box to search for whatever you would like to learn about first.
+All stakeholders, including researchers, software engineers, project leaders and funding teams, are encouraged to use _The Turing Way_ to understand their roles and responsibility of reproducibility in data science.
+You can inspect our resources on [GitHub](https://github.com/alan-turing-institute/the-turing-way), contribute to the project as described in our [contribution guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) and re-use all materials ([see the License](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md)).
```

Comment:
Love the way this is worded.

Reply:
Thanks for the feedback, Danny. I moved the links that you had highlighted in the opening line here to avoid linking out at the beginning of the paragraph.